Bottom: 6a3cc6d8f9f3245ba495aa10a17dc125519948f1
Top:    39fd3eb52090db407b562da1898fac96cc117e54
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-05-09 14:30:41 +0800

Refresh of vhost_net-conditionally-start

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 686bc07..0390432 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -354,7 +354,6 @@ static int vhost_net_tx_get_vq_desc(struct vhost_net *net,
 	return r;
 }
 
-
 static bool vhost_exceeds_maxpend(struct vhost_net *net)
 {
 	struct vhost_net_virtqueue *nvq = &net->vqs[VHOST_NET_VQ_TX];
